scripts/ci/deploy.sh

```diff
@@ -19,18 +19,37 @@
 set -e
```

Comment:
Right. Except that our production builds are code-signed, which the CI-servers don't currently do? Or am I misremembering?